app/main.py

```diff
@@ -39,10 +45,13 @@ def process(self, encrypted_jwt, tx_id=None):
 
         try:
             decrypted_payload = self._decrypt(encrypted_jwt, tx_id)
+            logger.debug("Decrypted message", tx_id=tx_id)
```

Comment:
Should log what you're about to do in preference to what you've just done in case the thing you do dies.